site2/docs/io-quickstart.md

```diff
@@ -26,18 +24,42 @@ the commands used in this tutorial can be used in a multi-node Pulsar cluster wi
 
 Before connecting Pulsar to a database, you need to install Pulsar and the desired built-in connector.
 
-For more information about **how to install a standalone Pulsar and built-in connectors**, see [here](getting-started-standalone.md/#installing-pulsar).
+Read the [Run a standalone Pulsar cluster locally](getting-started-standalone.md) for downloading the Pulsar distribution.
```

Comment:
Can you attach the preview screenshots for lines 27- 50 in the PR description?

Reply:
<img width="1728" alt="image" src="https://user-images.githubusercontent.com/18818196/188823205-fec8afa5-88c4-43ca-8ac7-6c0ca0e95f71.png">